src/components/common/TextDropdown.tsx

```diff
@@ -0,0 +1,30 @@
+import * as React from 'react'
+import { Text } from 'react-native'
+import { AirshipBridge } from 'react-native-airship'
+
+import { nightText } from '../../styles/common/textStyles'
+import { cacheStyles, Theme, useTheme } from '../services/ThemeContext'
+import { AirshipDropdown } from './AirshipDropdown'
+
+interface Props {
+  bridge: AirshipBridge<void>
+  message: string
+}
+
+export const TextDropdown = ({ bridge, message }: Props) => {
+  const theme = useTheme()
+  const styles = getStyles(theme)
+  return (
+    <AirshipDropdown bridge={bridge} backgroundColor={theme.modal}>
+      <Text style={styles.text}>{message}</Text>
+    </AirshipDropdown>
+  )
+}
+
+const getStyles = cacheStyles((theme: Theme) => ({
+  text: {
+    // @ts-expect-error
+    ...nightText('row-center'),
```

Comment:
The `nightText` stuff is all deprecated. If you splat out its contents, you end up with:

```js
text: {
 color: THEME.COLORS.WHITE,
 fontFamily: THEME.FONTS.DEFAULT,
 fontSize: THEME.rem(1),
 textAlign: 'center',
 flexShrink: 1,
 padding: theme.rem(1)
}
```

We are using v1 theme stuff in here, but I believe the `<AirshipDropdown>` itself may be stuck on using v1 theming as well, in which case we kinda have to match it.

Reply:
Yep let's not do that, now uses `<EdgeText>`

![image](https://user-images.githubusercontent.com/4023066/215693447-3733f57a-b61c-4fe8-8461-bf3bd1b7ad04.png)